FlickrViewer/FlickrViewer/FlickrProfileRequest.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  FlickrProfileRequest.swift
+//  FlickrViewer
+//
+//  Created by Kirill Shteffen on 18/07/2018.
+//  Copyright © 2018 BlackBricks. All rights reserved.
+//
+
+import Foundation
+
+class FlickrProfileRequest {
+    typealias FlickrResponse = (NSError?, User?) -> Void
+    
+    struct Keys {
+        static let flickrKey = "1ebbbfd26e664bd73f3dd4f88153e6e3"
+    }
+    
+    struct Errors {
+        static let invalidAccessErrorCode = 100
+    }
+    
+    class func fetchProfileForRequest( userId: String, onCompletion: @escaping FlickrResponse) -> Void {
+        print("START FETCHING PROFILE")
+        let urlString: String = "https://api.flickr.com/services/rest/?method=flickr.profile.getProfile&api_key=1ebbbfd26e664bd73f3dd4f88153e6e3&user_id=\(userId)&format=json&nojsoncallback=1"
```

Comment:
1ebbbfd26e664bd73f3dd4f88153e6e3 should be extracted into constant

and 
https://api.flickr.com/services/rest/?method=flickr.profile.getProfile&api_key=1ebbbfd26e664bd73f3dd4f88153e6e3&user_id=\(userId)&format=json&nojsoncallback=1
should be refactored by using "String Interpolation" or by using Alamofire framework